extract_unique_sensors_01.py

```diff
@@ -16,7 +19,9 @@ def create_parser():
     parser.add_argument('--url', help='A reference to SensorList.txt that specifies its ' 
                         'location on a computer network.') 
     parser.add_argument('--kat_sensor', required=True, help='Name of unique sensor')
+    
     args = parser.parse_args()
+    logging.basicConfig(format='%(message)s', level=logging.ERROR, stream=sys.stdout)
```

Comment:
I see that you configured the logging inside a function and happen to use logging in another function...
I am getting a headache trying to wrap my head around this implementation...

Let's have a chat later today.

Reply:
Yes, please. Thank you.

Reply:
It seems to work either way, but I have configured the logging inside the main function. It seems to be the most correct way.